content/docs/command-reference/exp/diff.md

```diff
@@ -0,0 +1,69 @@
+# exp diff
+
+Show changes in [metrics](/doc/command-reference/metrics) and
+[parameters](/doc/command-reference/params) between `dvc experiments`.
+
+## Synopsis
+
+```usage
+usage: dvc experiments diff [-h] [-q | -v] [--all] [--show-json]
+                            [--show-md] [--old] [--no-path]
+                            [--precision <n>] [a_rev] [b_rev]
+
+positional arguments:
+  a_rev            Old experiment to compare (defaults to HEAD)
+  b_rev            New experiment to compare (defaults to the current
+                   workspace)
+```
+
+# Description
+
+Provides a quick way to compare `dvc params` and `dvc metrics` between two
+`dvc experiments` by printing a table of differences. It includes the
+params/metrics file "Path", "Param" or "Metric" name, their "Old" and "New"
+values, and the difference ("Change") if both values are numbers.
+
+> This is similar to combining the reports from `dvc params diff` and
+> `dvc metrics diff` together, for the experiments in question.
+
+Without arguments, this command compares all the params and metrics currently
+present in the <abbr>workspace</abbr> (e.g. the last `dvc exp run`) with the
+latest committed versions (required). Only params/metrics that changed are
+listed, by default (show everything with `--all`).
+
+`a_rev` and `b_rev` are optional experiments to compare. They accept experiment
+names or hashes (see `dvc exp run`). Providing a single one results in comparing
+it to the workspace.
+
+All params and metrics defined in the experiments' `dvc.yaml` files are used by
+default.
```

Comment:
Why no `--targets` in this command? Could accept both params and metrics.

Reply:
The only reason is because metrics and params also did not accept `--targets` at the time `exp diff` was implemented. It can be added as needed.

Reply:
We can discuss in #5530. Thanks